script/server

```diff
@@ -3,4 +3,5 @@
 set -e
 cd "$(dirname "$0")/.."
 
-script/fixmystreet_app_server.pl --fork --restart --debug $@
+export FIXMYSTREET_APP_DEBUG=1
```

Comment:
Could this be conditional on `FIXMYSTREET_APP_DEBUG` not already being defined? It'd be handy to be able to disable the toolbar by running `FIXMYSTREET_APP_DEBUG=0 ./script/server`.